src/sd_api_v2/sdk/components/serialization/application/codecs/common/conn_systemreset.h

```diff
@@ -66,7 +66,7 @@ extern "C" {
  * @retval NRF_SUCCESS          Encoding success.
  * @retval NRF_ERROR_INTERNAL   Encoding failure. Transport error.
  */
-uint32_t conn_systemreset(void);
+// uint32_t conn_systemreset(void);
```

Comment:
Commented out code with documentation is not a good idea, remove.

Reply:
This file is now an empty file actually. It was reserved for the reset function and it is now reserved for refactoring the code.